packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts

```diff
@@ -185,21 +180,51 @@ export class ModelViewerGLTFInstance extends GLTFInstance {
       clone.normalMap = material.normalMap.clone();
       clone.normalMap.needsUpdate = true;
     }
-    if (material.aoMap != null) {
-      clone.aoMap = material.aoMap.clone();
-      clone.aoMap.needsUpdate = true;
-    }
     if (material.emissiveMap != null) {
       clone.emissiveMap = material.emissiveMap.clone();
       clone.emissiveMap.needsUpdate = true;
     }
-    if (material.metalnessMap != null) {
+
+    // Clones the roughnessMap if it exists.
+    let roughnessMap: Texture|null = null;
+    if (material.roughnessMap) {
+      roughnessMap = material.roughnessMap!.clone();
+    }
+
+    // Assigns the roughnessMap to the cloned material and generates mipmaps.
+    if (roughnessMap !== null) {
```

Comment:
This is the one time where we prefer `==` and `!=`, since they catch both null and undefined. There are lots of opinions about it, but that's model-viewer style.

Reply:
I'm not sure what you're suggesting. Do you want a != check to see if it can be coerced into null?
Added != null check.

Reply:
no, instead of, not in addition to. We just don't use `=== null` or `!== null`.